web/src/features/charts/bar-breakdown/elements/BarBreakdownAxisLegend.tsx

```diff
@@ -0,0 +1,51 @@
+import { FaArrowLeft, FaArrowRight } from 'react-icons/fa';
```

Comment:
```suggestion
import { FaArrowLeft, FaArrowRight } from 'react-icons/fa6';
```
Not sure it matters but we are importing most other fa icons from fa6. Not sure the arrow design is different though.